### PR TITLE
Metric validation

### PR DESCRIPTION
**Pull request by mgupta0421**
This PR implements the following functionalities -
1- Query results for multiple query search will be displayed in results table.
2- Download results to excel file.
3- Metrics validation errors handled in UI.
4- Added limit option for the users to select the number of results to be displayed
5- Completed validation for metric in the query

### CODE REVIEW

File: frontend/src/components/landingPage/Query.vue [hunk 1]
```diff
@@ -11,24 +11,54 @@
     ></textarea>
     <div id="errorDiv">
       <div
-        class="error"
+        class="errorr"
```
Comment (bhermann) [inline frontend/src/components/landingPage/Query.vue]: Is this right?

File: frontend/src/components/landingPage/SearchPart.vue [hunk 2]
```diff
@@ -60,15 +66,40 @@
...
+            <v-spacer></v-spacer>
+            <v-btn color="green darken-1" text @click="dialog = false">No</v-btn>
+            <v-btn color="green darken-1" text @click="download">Yes</v-btn>
+          </v-card-actions>
+        </v-card>
+      </v-dialog>
+
+      <v-dialog v-model="dialog2" max-width="550">
+        <v-card>
+          <v-card-title class="headline">OOPS...!!!</v-card-title>
```
Comment (bhermann) [inline frontend/src/components/landingPage/SearchPart.vue]: OOPS?

File: frontend/src/components/landingPage/Query.vue (hunk 1 above)
Comment (SViktorS) [reply frontend/src/components/landingPage/Query.vue]: Fixed it.

File: frontend/src/components/landingPage/SearchPart.vue (hunk 2 above)
Comment (SViktorS) [reply frontend/src/components/landingPage/SearchPart.vue]: Fixed it.

**Pull request merged**